### PR TITLE
feat(about-page): microchips section

### PR DESCRIPTION
**Pull request by im-klau**
## Jira issue

JIRA issue: [FSU-153](https://klau.atlassian.net/browse/FSU-153)

## Describe your changes
This PR introduces the Microchip section alongside with the promo form.

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/frontend/app/templates/About/components/Microchips/Microchips.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        <picture>
+          <source
+            srcSet="/about/safe-animal-desktop.png"
+            media="(min-width: 1200px)"
+          />
+          <source
+            srcSet="/about/safe-animal-tablet.png"
+            media="(min-width: 587px)"
+          />
+          <Styled.Image src="/about/safe-animal-mobile.png" alt="" />
```
Comment (AmandaKr) [inline packages/frontend/app/templates/About/components/Microchips/Microchips.jsx:30]: alt is missing

**Review by AmandaKr — COMMENTED** [review]
image
This image has black parts of the left and right 🤔

**Comment by AmandaKr** [comment]
It looks good for me, but on design tis form has lager background
image
design:
image

File: packages/frontend/app/templates/About/components/Microchips/Microchips.jsx (hunk 1 above)
Comment (im-klau) [reply packages/frontend/app/templates/About/components/Microchips/Microchips.jsx:30]: Well, actually this is a nice topic as empty alt is 100% correct when it comes to HTML and accessibility :D 

The thing is that empty alt should be used only with decorative images, but if the image is really important, then it should be well described.

**Comment by im-klau** [comment]
> It looks good for me, but on design this form has lager background image design: image

The design has changed recently, it doesn't have a full width background anymore 👀

**Review by AmandaKr — APPROVED** [review]
Well done, only this image has black margins on some devices

**Review by Ebmili — APPROVED** [review]
Looks great :)

**Comment by im-klau** [comment]
> Well done, only this image has black margins on some devices image

Yeah, unfortunately, there's no other way to add such a banner with a fixed height so that the copy within it will be always visible 😢

**Pull request merged**